data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/BulkRetryStrategy.java

```diff
@@ -248,7 +255,13 @@ private BulkOperationRequestResponse handleRetriesAndFailures(final Accumulating
     }
 
     private void handleFailures(final AccumulatingBulkRequest<BulkOperationWrapper, BulkRequest> bulkRequest, final BulkResponse bulkResponse, final Throwable failure) {
+        LOG.warn("Bulk Operation Failed.", failure);
```

Comment:
Do we want the full stack trace here? We know where the exception is thrown from right?

Or should this be the following?

```
 LOG.warn("Bulk Operation Failed. {}", failure.getMessage());
```

Reply:
This function may be called from multiple paths, right?